book/src/bridge/extern_rustqt.md

```diff
@@ -131,7 +131,19 @@ Where `<Property>` is the name of the property.
 
 These setters and getters assure that the changed signal is emitted every time the property is edited.
 
-> Note that in the future it will be possible to specify custom getters and setters
+It is also possible to specify custom getters, setters and on-changed functions, using flags passed like so:
+`#[qproperty(TYPE, NAME, read = myGetter, write = mySetter, notify = myOnChanged)]`
+
+> Note: currently TypeName lookups are not currently supported, so either the name must be camel case or specified like `#[cxx_name = "my_getter"]`
```

Comment:
This is written from the perspective of a developer who knows what the TypeName struct is (so pretty much you, me and Andrew :wink: )
A user who reads this documentation won't know what this means.

Reply:
Done